todo_project_name/rsa.py

```diff
@@ -72,11 +98,13 @@ def save_key(key: RSAKey, path: Path) -> Path:
 def read_key(path: Path, key_type: Type[RSAKeyVar]) -> RSAKeyVar:
     """Read RSA key from the file."""
     with path.open("r", encoding="utf8") as file:
+        # Skip the heading.
```

Comment:
perhaps we could check if heading matches type of key we are supposed to be reading

Reply:
Yes, I think that is a good idea. I haven't thought about it in the first implementation a few pushes ago, so I simply didn't change it this time (just added a comment). I would defer the change until we establish the format, if any. The change would be within the same function, so I don't think some “todo” comment is necessary.